tests/checks/fish_config.fish

```diff
@@ -19,7 +19,7 @@ fish_config theme show "fish default"
 # CHECK: {{\x1b\[m}}echo{{\x1b\[m}} 'Errors are the portal to discovery
 # CHECK: {{\x1b\[m}}Th{{\x1b\[m}}is an autosuggestion
 # CHECK: {{\x1b\[m}}{{\x1b\[4m}}fish default{{\x1b\[m}}
-# CHECK: {{\x1b\[m}}/bright/vixens{{\x1b\[m}} {{\x1b\[36m}}jump{{\x1b\[m}} {{\x1b\[36m}}{{\x1b\[1m}}|{{\x1b\[m}} {{\x1b\[33m}}"fowl"{{\x1b\[m}} {{\x1b\[36m}}{{\x1b\[1m}}> quack{{\x1b\[m}} {{\x1b\[32m}}&{{\x1b\[m}}{{\x1b\[31m}} # This is a comment
+# CHECK: {{(?:\x1b\[m)?}}/bright/vixens{{\x1b\[m}} {{\x1b\[36m}}jump{{\x1b\[m}} {{\x1b\[36m}}{{\x1b\[1m}}|{{\x1b\[m}} {{\x1b\[33m}}"fowl"{{\x1b\[m}} {{\x1b\[36m}}{{\x1b\[1m}}> quack{{\x1b\[m}} {{\x1b\[32m}}&{{\x1b\[m}}{{\x1b\[31m}} # This is a comment
```

Comment:
yeah this test is flaky in CI but I don't think this is enough
to fix it, see https://cirrus-ci.com/task/4724192307838976?logs=tests#L280-L301
Until that's fixed, maybe we should filter output with something like `SANITIZE_FOR_PRINTING_RE`.
I'd prefer if we try to deal with this test in a separate PR, since it's not so easy.